fortitude/src/cli.rs

```diff
@@ -145,4 +106,96 @@ pub struct CheckArgs {
     /// Options are "off" (default), "ascii", and "fancy"
     #[arg(long, value_enum)]
     pub progress_bar: Option<ProgressBar>,
+
+    // Rule selection
+    /// Comma-separated list of rules to ignore.
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "RULE_CODE",
+        value_parser = RuleSelectorParser,
+        help_heading = "Rule selection",
+        hide_possible_values = true
+    )]
+    pub ignore: Option<Vec<RuleSelector>>,
+
+    /// Comma-separated list of rule codes to enable (or ALL, to enable all rules).
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "RULE_CODE",
+        value_parser = RuleSelectorParser,
+        help_heading = "Rule selection",
+        hide_possible_values = true
+    )]
+    pub select: Option<Vec<RuleSelector>>,
+
+    /// Like --select, but adds additional rule codes on top of those already specified.
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "RULE_CODE",
+        value_parser = RuleSelectorParser,
+        help_heading = "Rule selection",
+        hide_possible_values = true
+    )]
+    pub extend_select: Option<Vec<RuleSelector>>,
+
+    /// List of mappings from file pattern to code to exclude.
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "FILE_PATTERN:RULE_CODE",
+        help_heading = "Rule selection"
+    )]
+    pub per_file_ignores: Option<Vec<PatternPrefixPair>>,
+
+    /// Like `--per-file-ignores`, but adds additional ignores on top of those already specified.
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "FILE_PATTERN:RULE_CODE",
+        help_heading = "Rule selection"
+    )]
+    pub extend_per_file_ignores: Option<Vec<PatternPrefixPair>>,
+
+    // File selection
+    /// File extensions to check
+    #[arg(
+        long,
+        value_delimiter = ',',
+        default_values = FORTRAN_EXTS,
+        help_heading = "File selection"
+    )]
+    pub file_extensions: Option<Vec<String>>,
+
+    /// List of paths, used to omit files and/or directories from analysis.
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "FILE_PATTERN",
+        help_heading = "File selection"
+    )]
+    pub exclude: Option<Vec<FilePattern>>,
+
+    /// Like --exclude, but adds additional files and directories on top of those already excluded.
+    #[arg(
+        long,
+        value_delimiter = ',',
+        value_name = "FILE_PATTERN",
+        help_heading = "File selection"
+    )]
+    pub extend_exclude: Option<Vec<FilePattern>>,
+
+    /// Enforce exclusions, even for paths passed to Ruff directly on the command-line.
```

Comment:
```suggestion
    /// Enforce exclusions, even for paths passed to Fortitude directly on the command-line.
```

Reply:
Whoops! It was only a matter of time before I let one of those slip through